File: samples/server/petstore/jaxrs-resteasy/eap-java8/src/main/java/io/swagger/api/impl/UserApiServiceImpl.java
package io.swagger.api.impl;

import io.swagger.api.*;
import io.swagger.model.*;


import java.util.List;
import io.swagger.model.User;

import java.util.List;

import java.io.InputStream;

import javax.ws.rs.core.Response;
import javax.ws.rs.core.SecurityContext;


public class UserApiServiceImpl implements UserApi {
      public Response createUser(User body,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response createUsersWithArrayInput(List<User> body,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response createUsersWithListInput(List<User> body,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response deleteUser(String username,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response getUserByName(String username,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response loginUser(String username,String password,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response logoutUser(SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
      public Response updateUser(String username,User body,SecurityContext securityContext) {
      // do some magic!
      return Response.ok().build();
  }
}

File: samples/server/petstore/kotlin-springboot-reactive/src/main/kotlin/org/openapitools/api/UserApiServiceImpl.kt
package org.openapitools.api

import org.openapitools.model.User
import kotlinx.coroutines.flow.Flow;
import org.springframework.stereotype.Service
@Service
class UserApiServiceImpl : UserApiService {

    override suspend fun createUser(body: User): Unit {
        TODO("Implement me")
    }

    override suspend fun createUsersWithArrayInput(body: Flow<User>): Unit {
        TODO("Implement me")
    }

    override suspend fun createUsersWithListInput(body: Flow<User>): Unit {
        TODO("Implement me")
    }

    override suspend fun deleteUser(username: kotlin.String): Unit {
        TODO("Implement me")
    }

    override suspend fun getUserByName(username: kotlin.String): User {
        TODO("Implement me")
    }

    override suspend fun loginUser(username: kotlin.String, password: kotlin.String): kotlin.String {
        TODO("Implement me")
    }

    override suspend fun logoutUser(): Unit {
        TODO("Implement me")
    }

    override suspend fun updateUser(username: kotlin.String, body: User): Unit {
        TODO("Implement me")
    }
}
